pylint/checkers/typecheck.py

```diff
@@ -21,6 +21,7 @@
 from typing import TYPE_CHECKING, Any, Union
 
 import astroid.exceptions
```

Comment:
```suggestion

import astroid
import astroid.exceptions
```
Sorry for the confusion here. I missed that the bare `astroid` import is used as well.